### ISSUE TITLE
CLI version arguments display nothing

### ISSUE THREAD

**Issue by ParticleMon** [wwmm/easyeffects#3300]
### EasyEffects Version

7.1.6 

### What package are you using?

Other (specify below)

### Distribution

Linux Mint 22

### Describe the bug

`easyeffects -v` and `easyeffects --version` display nothing.

### Expected Behavior

The version number is displayed.

**Comment by wwmm** [comment]
It seems a regression happened somewhere.

**Comment by AlexFolland** [comment]
I can't reproduce this here.  Here's the output on my system.

```Plaintext
[alex@alex-pc ~]$ easyeffects -v

(easyeffects:28536): Gdk-CRITICAL **: 14:40:51.949: gdk_monitor_set_scale: assertion 'scale >= 1.' failed

(easyeffects:28536): Adwaita-WARNING **: 14:40:52.400: Using GtkSettings:gtk-application-prefer-dark-theme with libadwaita is unsupported. Please use AdwStyleManager:color-scheme instead.

(easyeffects:28536): easyeffects-WARNING **: 14:40:52.621: 	rnnoise.cpp:366sie:  empty model name set, using the standard model
easyeffects version: 7.1.7
Segmentation fault (core dumped)
[ble: exit 139]
[alex@alex-pc ~]$
```

**Comment by wwmm** [comment]
>I can't reproduce this here. Here's the output on my system.

I think I understand what is going on. The messages are being printed with a function that only does something in debug mode.

**Comment by wwmm** [comment]
> > I can't reproduce this here. Here's the output on my system.
> 
> I think I understand what is going on. The messages are being printed with a function that only does something in debug mode.

Actually that was not the case. The version is being printed by a simple `std::cout` and somehow it is working again on my computer without any code change. I wonder if the gtk4 update somehow fixed it.

As things seems to be fine I will close this issue.

**Comment by ParticleMon** [comment]
> somehow it is working again on my computer without any code change. I wonder if the gtk4 update somehow fixed it.
> 
> As things seems to be fine I will close this issue.

It's not suddenly working for me. I've not received (or noticed) a GTK 4 update.

**Comment by wwmm** [comment]
> > somehow it is working again on my computer without any code change. I wonder if the gtk4 update somehow fixed it.
> > As things seems to be fine I will close this issue.
> 
> It's not suddenly working for me. I've not received (or noticed) a GTK 4 update.

Ok. For now let's reopen this issue.

**Comment by wwmm** [comment]
@ParticleMon I did more tests and I think this time I've found the source fo the problem. Gtk has two callbacks to handle command line arguments but only one of them actually works when an instance is already running in the background. When you have time test the command again after killing easyEffects. It will probably work.

I will update our master branch with a fix for this. Moving the option parsing to the other callback will solve the problem.
